### PR TITLE
Finished the HabitTracker

### PR DESCRIPTION
**Pull request by Deviationlark**
Had fun doing this while learning sql, even though i'll have to do the app again to reinforce my knowledge but i still want to get a review before making it to see what i could do better.

### CODE REVIEW

**Review by Doc415 — CHANGES_REQUESTED** [review]
Can you please fix the codacy warning

**Comment by Deviationlark** [comment]
Its all readme issues on codacy

File: HabitTracker.Deviationlark/Habit-Tracker/Menu.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                switch (input)
+                {
+                    case "0":
+                        Console.WriteLine("Goodbye!");
+                        Console.ReadLine();
+                        closeApp = true;
+                        Environment.Exit(0);
+                        break;
+                    case "1":
+                        CRUD.GetAllRecords();
```
[the remark this answers is not in the document]
Comment (Doc415) [reply HabitTracker.Deviationlark/Habit-Tracker/Menu.cs]: I ll see it if i got a working code :))

**Review by Doc415 — APPROVED** [review]
Very good work!
Some suggestions:
After selecting habit to view app shows the records and waits a user input . It feels like should enter the id but it is just waiting. After presing enter it asks for the id. User can be prompted like "Press enter to continue"

You have a validation for negative numbers , and it ll be good the also for 0 . I should not be able to enter 0 pages for my reading habit.

After selecting existing habit, user cannot return to first menu to add or remove new habits. That would be nice to have that option.

I am approving your project.

**Comment by Deviationlark** [comment]
why shouldn't you be able to enter 0 pages? if you didn't read that day why not enter 0 pages?

**Pull request closed without merging**